arangodb-net-standard/ApiClientBase.cs

```diff
@@ -43,15 +51,29 @@ protected void ValidateDocumentId(string documentId)
 
         protected T DeserializeJsonFromStream<T>(Stream stream)
         {
-            return _serialization.DeserializeFromStream<T>(stream);
+            try
+            {
+                return _serialization.DeserializeFromStream<T>(stream);
+            }
+            catch (Exception e)
+            {
+                throw new SerializerException($"An error occured while Deserializing the data response from Arango. See InnerExpection for more details.", e);
+            }
         }
 
         protected byte[] GetContent<T>(T item, bool useCamelCasePropertyNames, bool ignoreNullValues)
         {
-            return _serialization.Serialize<T>(
-                item,
-                useCamelCasePropertyNames,
-                ignoreNullValues);
+            try
+            {
+                return _serialization.Serialize<T>(
+                    item,
+                    useCamelCasePropertyNames,
+                    ignoreNullValues);
+            }
+            catch (Exception e)
+            {
+                throw new SerializerException($"A serialization error occured while preparing a request for Arango. See InnerExpection for more details.", e);
```

Comment:
There is a small typo here: `InnerExpection` should be `InnerException`. Same in other messages.